app/app.go

```diff
@@ -491,7 +528,8 @@ func NewIxoApp(
 // MakeCodecs constructs the *std.Codec and *codec.LegacyAmino instances used by
 // ixoapp. It is useful for tests and clients who do not want to construct the
 // full ixoapp.
-func MakeCodecs() (codec.Marshaler, *codec.LegacyAmino) {
+// TODO: Dont know if this Codec is correct
+func MakeCodecs() (codec.Codec, *codec.LegacyAmino) {
```

Comment:
@ig-shaun @demondayza @steveixo 
Same as previous comment related to the Codec changes.